cookbooks/aws-parallelcluster-platform/kitchen.platform-install.yml

```diff
@@ -96,6 +96,22 @@ suites:
     verifier:
       controls:
         - /tag:install_cron/
+  - name: spack
+    run_list:
+      - recipe[aws-parallelcluster-tests::setup]
+      - recipe[aws-parallelcluster-tests::test_resource]
+    verifier:
+      controls:
+        - /tag:spack/
```

Comment:
the convention right now is to use the tag in one of two ways, either with _install or _config based on whether it is baked into the AMI or run at boot.  If you're test doesn't use either of those yet, just use a name without `tag:`